### PR TITLE
data: sahil-sagwekar2652

### PR DESCRIPTION
**Pull request by sahil-sagwekar2652**
## Note to reviewers
This pull request does not fix anything. It is for my LinkFree page

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5220"></a>

### CODE REVIEW

File: data/sahil-sagwekar2652.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		},
+		{
+			"name": "YouTube",
+			"url": "https://www.youtube.com/@SahilSagwekar",
+			"icon": "FaYouTube"
+		},
+		{
+			"name": "LinkedIn",
+			"url": "https://www.linkedin.com/in/sahil-sagwekar-0b955b223/",
+			"icon": "FaLinkedIn"
```
Comment (priyankarpal) [inline data/sahil-sagwekar2652.json:18]: change the icon from `FaLinkedIn` to `SiLinkedin`
``` suggestion
			"icon": "SiLinkedin"
```

File: data/sahil-sagwekar2652.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		},
+		{
+			"name": "GitHub",
+			"url": "https://github.com/sahil-sagwekar2652",
+			"icon": "FaGitHub"
+		},
+		{
+			"name": "My Hashnode blog",
+			"url": "https://sagwekarsahil2652.hashnode.dev",
+			"icon": "FaHashnode"
```
Comment (priyankarpal) [inline data/sahil-sagwekar2652.json:28]: Also here change the hashnode icon to `SiHashnode`
``` suggestion
			"icon": "SiHashnode"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
I added single-line comments.

**Review by priyankarpal — APPROVED** [review]
Now it's looking great. Ready to merge it. 🎉

**Review by SurajPratap10 — APPROVED** [review]

**Review by iamthecloverly — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**